.circleci/config.yml

```diff
@@ -511,14 +493,18 @@ jobs:
     resource_class: xlarge
     steps:
       - checkout
+      - <<: *markJobStartsOnGCS
       - run:
           command: |
-            mkdir -p /go/out/racetests
-            trap "go-junit-report </go/out/racetests/go-racetest-report.out > /go/out/racetests/go-racetest-report.xml" EXIT
+            mkdir -p /go/out/tests
```

Comment:
check with @xiaolanz if this is OK.

Reply:
Yes, just did, and she is happy with it.